user-interface/src/main/java/life/qbic/datamanager/views/register/UserRegistrationMain.java

```diff
@@ -81,12 +81,12 @@ private void handleRegistrationFailure(List<RuntimeException> exceptionList) {
     if (exceptionList.isEmpty()) {
       return;
     }
-    if (exceptionList.contains(UserExistsException.class)) {
+    if (exceptionList.stream().anyMatch(e -> e instanceof UserExistsException)) {
```

Comment:
why the changes?

Reply:
In a nutshell the contains method does not work for instance based comparison in generics. Since we're only interested in the type of exception thrown, this had to be adapted.

Reply:
Nice catch. However please convert it to a for loop. 
```java
if (myList.isEmpty()) {
  return;
}
for (RuntimeExceptione : myList) {
  if (e instanceof UserExistsException userExistsException) {
   //... do stuff with userExistsException
  } else if (e instanceof OtherException otherException) {
   //... do stuff with otherException
  } else if (...) {
   //... do stuff with ... exception
  }
}
return;

```

Reply:
That's a good point! Thanks for the suggestion <3